src/test/kotlin/ru/hse/dynamomock/AWSDynamoDBMockTest.kt

```diff
@@ -121,8 +234,12 @@ internal open class AWSDynamoDBMockTest {
         ) + attributeDefinitionPool.indices.flatMap { i ->
             listOf(
                 createTableMetadata("TEST_N_$i", i, null, Instant.ofEpochMilli(123241424222 * i)),
-                createTableMetadata("TEST_M_$i", i, i, Instant.ofEpochMilli(9472938474 * i + 2))
             )
         }
+
+        @Container
+        val dynamoDb = GenericContainer("amazon/dynamodb-local:1.13.2")
+            .withCommand("-jar DynamoDBLocal.jar -inMemory -sharedDb")
+            .withExposedPorts(8000)
```

Comment:
Нет ли возможности использовать рандомный порт + проверять, не занят ли он? Кажется припоминаю что такое можно было.

Reply:
В документации сказано следующее: 
Note that this exposed port number is from the perspective of the container.

_From the host's perspective Testcontainers actually exposes this on a random free port. This is by design, to avoid port collisions that may arise with locally running software or in between parallel test runs._

То есть порт действительно рандомный. И если он нужен, то его нужно получать в рантайме вот так вот: `Integer firstMappedPort = container.getMappedPort(8080);`